lib-ee/emqx_ee_bridge/i18n/emqx_ee_bridge_kafka.conf

```diff
@@ -62,7 +62,7 @@ emqx_ee_bridge_kafka {
     mqtt_topic {
         desc {
             en: "MQTT topic or topic as data source (bridge input).  Should not configure this if the bridge is used as a rule action."
```

Comment:
```suggestion
            en: "MQTT topic or topic filter as data source (bridge input).  If rule action is used as data source, this config should be left empty, otherwise messages will be duplicated in Kafka."
```